lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -189,21 +209,37 @@ class MongoClientInterface {
         return this.db.collection(newName);
     }
 
+    /**
+     * Creates a bucket with the provided metadata
+     * @param {string} bucketName bucket name
+     * @param {Object} bucketMD bucket metadata
+     * @param {Object} log logger
+     * @param {Function} cb callback
+     * @return {undefined}
+     */
     createBucket(bucketName, bucketMD, log, cb) {
         // FIXME: there should be a version of BucketInfo.serialize()
         // that does not JSON.stringify()
         const bucketInfo = BucketInfo.fromObj(bucketMD);
         const bucketMDStr = bucketInfo.serialize();
         const newBucketMD = JSON.parse(bucketMDStr);
         const m = this.getCollection(METASTORE);
+
+        const payload = {
+            _id: bucketName,
+            value: newBucketMD,
+        };
+        if (bucketName !== constants.usersBucket &&
+            bucketName !== PENSIEVE &&
+            !bucketName.startsWith(constants.mpuBucketPrefix)) {
+            payload.vFormat = this.defaultBucketKeyFormat;
```

Comment:
generally (and thinking about typing/upcoming typescript migration) : vFormat should be initialized anyway.
if we need an extra different behavior we can introduce yet another value (like "unversioned")